src/pages/ServiceDetails/ServiceMetrics.tsx

```diff
@@ -59,12 +59,12 @@ class ServiceMetrics extends React.Component<ServiceId, ServiceMetricsState> {
   };
 
   onPollIntervalChanged = (pollInterval: number) => {
-    let newRefInterval: NodeJS.Timer | undefined = undefined;
+    let newRefInterval: number | undefined = undefined;
     if (this.state.pollMetrics) {
       clearInterval(this.state.pollMetrics);
```

Comment:
maybe here also `window.clearInterval` would avoid confusion, or is it not necessary?

Reply:
There wasn't any warning or error with that, i'll double check and maybe it wouldn't hurt to call it like that.